### PR TITLE
Created a superuser and a database to hold blog posts. Created templa…

### PR DESCRIPTION
**Pull request by teganjennings**
Created a DB to hold blog posts. Created templates. Created an superuser

### CODE REVIEW

**Comment by vfxGer** [comment]
.pyc files should not be in here. Did you manage to edit your global gitignore? Even just have a .gitignore file for this repo would ok too.

**Review by vfxGer — CHANGES_REQUESTED** [review]
.pyc files should not be in here

File: django_project/blog/admin.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
+from .models import Post
 
-# Register your models here.
+admin.site.register(Post)
```
Comment (vfxGer) [inline django_project/blog/admin.py:4]: it's good practice to have a new line at the end of a file, not a a major concern but good practice

Comment (vfxGer) [reply django_project/blog/admin.py:4]: Here's a link that explains it better than I could https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

File: django_project/blog/models.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
 
-# Create your models here.
+class Post(models.Model):
+    title = models.CharField(max_length=100)
+    content = models.TextField()
+    date_posted = models.DateTimeField(default=timezone.now)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+
+    def __str__(self):
+        return self.title
```
Comment (vfxGer) [inline django_project/blog/models.py:12]: again add a new line

File: django_project/blog/templates/blog/base.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
+{% load static %}
+<html>
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <!-- Bootstrap CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```
Comment (vfxGer) [inline django_project/blog/templates/blog/base.html:9]: FYI we would generally download things like this and put them in our repo but this is fine

File: django_project/blog/models.py (hunk 2 above)
Comment (teganjennings) [reply django_project/blog/models.py:12]: Ive added the new line on every file that didn't have it.

File: django_project/blog/admin.py (hunk 1 above)
Comment (teganjennings) [reply django_project/blog/admin.py:4]: That makes sense, Ive added it.

File: django_project/blog/templates/blog/base.html (hunk 3 above)
Comment (teganjennings) [reply django_project/blog/templates/blog/base.html:9]: Downloaded and changed scripts

**Review by teganjennings — COMMENTED** [review]
Finished changes

**Comment by vfxGer** [comment]
Don't merge until you get a reviewer to approve it. I don't see your fixes?

**Comment by teganjennings** [comment]
> Don't merge until you get a reviewer to approve it. I don't see your fixes?

Sorry. I made them and pushed them in the second commit on the PR? 
I think i've reverted this PR and will do the changes again and push

**Comment by teganjennings** [comment]
> Don't merge until you get a reviewer to approve it. I don't see your fixes?

Sorry. I made them and pushed them in the second commit on the PR? 


> .pyc files should not be in here. Did you manage to edit your global gitignore? Even just have a .gitignore file for this repo would ok too.

Ive edited it but im not sure if it will kick into this repo because it was created before i edied the gitignore.global

**Pull request merged**